run_project_tests.py

```diff
@@ -614,6 +623,10 @@ def gather_tests(testdir: Path, stdout_mandatory: bool) -> T.List[TestDef]:
         if test_def_file.is_file():
             test_def = json.loads(test_def_file.read_text())
 
+        # Perform setup for boost tests that require absolute paths in nativefile.ini
+        if str(t.path).endswith('32 boost root') or str(t.path).endswith('33 boost split root'):
```

Comment:
This should work for all tests if the `nativefile.ini.in` is present, not just boost.

Reply:
Fixed, see `detect_parameter_files()`